chunks/lang-python/Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN sudo install-packages \
 	libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev \
 	# Install PYENV
 	&& git clone https://github.com/pyenv/pyenv.git ~/.pyenv \
-	&& git -C ~/.pyenv checkout 0963c8ac02bca616b8f91ab8b1902f48502cd813 \
+	&& git -C ~/.pyenv checkout 9ecd803bffaffb949fbdd8c70cb086227f6a3202 \
```

Comment:
How about use tags? easier to read
```
git -C ~/.pyenv checkout v2.4.10
```

Reply:
Yes, but we use commits to pin an exact, unchangeable version. It's a security thing.

Reply:
And we could make it an autofix https://github.com/pyenv/pyenv/releases as well

Reply:
(For instance, GitHub actions [recommend](https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions) to do this)

Reply:
I've added something human-readable in [`723edf8` (#1402)](https://github.com/gitpod-io/workspace-images/pull/1402/commits/723edf80506cb92c95c44b71fe53f16c228d19ae)

Reply:
> Yes, but we use commits to pin an exact, unchangeable version. It's a security thing.

Then our repo can always has security problem, i.e: we use versions like `0.0.1` to download `bun` but not check it's checksum

Reply:
It's true that we use tags in many places, but it's more fine with 1st parties like bun, where we get it straight from the source. For our use-cases with dazzle chunks, it also is not always possible/practical to do, so it's a balancing act.

Reply:
FWIW, I don't think changing this here is a good idea (having the comments would make this easier to parse for humans and pinned for machines), and also don't consider this too high stakes to invest more time into changing the workflows in this repo.